thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -293,15 +297,20 @@ export default Ember.Controller.extend({
       this.setProperties({ hoverUrns: new Set(urns), hoverTimestamp: timestamp });
     },
 
+    onSessionChange(name, text) {
+      console.log('onSessionChange(name, text)');
```

Comment:
Does this need to stay for future debugging, or can we remove it for this PR?

Reply:
fixed